zproject/dev_settings.py

```diff
@@ -73,3 +73,6 @@
 # Two factor authentication: Use the fake backend for development.
 TWO_FACTOR_CALL_GATEWAY = 'two_factor.gateways.fake.Fake'
 TWO_FACTOR_SMS_GATEWAY = 'two_factor.gateways.fake.Fake'
+
+RECAPTCHA_SITEKEY = '6LdnpjsUAAAAALDmJgiuNoa7R-pyHSEHdpAuHBJp'
+RECAPTCHA_SECRET  = '6LdnpjsUAAAAAN9ZDg7wlYKcywU6CKplzsKUTZrz'
```

Comment:
Let's give these a comment like
```
# reCAPTCHA: Upstream docs for v1 (https://developers.google.com/recaptcha/old/intro)
# say to just use any keys for development. These were made by umair.waheed@gmail.com.

Reply:
Sure. I think using the test keys provided by Google is even better. Reference: https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha-v2-what-should-i-do

Reply:
If those work, great.

That documentation is for the non-invisible API, and other documentation suggests that a given key should work for only one of the APIs, not several. But we should be able to confirm that these work.

Reply:
Yep, they work.